springwolf-add-ons/springwolf-common-model-converters/src/main/java/io/github/stavshamir/springwolf/addons/common_model_converters/converters/monetaryamount/MonetaryAmount.java

```diff
@@ -10,7 +10,7 @@
 class MonetaryAmount {
 
     @JsonProperty("amount")
-    @Schema(example = "99.99")
+    @Schema(example = "99.99", minimum = "0.1")
```

Comment:
Any specific reason the minimum cannot be "0.0"?

Reply:
No reason, this demonstrates an new field in springwolf-ui
Changing this should be easy